mlflow/utils/cli_args.py

```diff
@@ -43,3 +43,10 @@
 # We use None to disambiguate manually selecting "4"
 WORKERS = click.option("--workers", "-w", default=None,
                        help="Number of gunicorn worker processes to handle requests (default: 4).")
+
+# default disable
+CONFIG = click.option("--config", "-c", default=None, metavar="FILE",
+                      help="The Gunicorn config file is a .py file (default: None). "
+                           "You'd better add the file by absolute root. "
```

Comment:
Maybe you could rephrase the help message: "Path to a gunicorn config file (default None). See https://docs.gunicorn.org/en/stable/configure.html for more info on the format of this option."

We should prefer to link to reference documentation than to blog posts.